docs/data/material/customization/container-queries/container-queries.md

```diff
@@ -0,0 +1,86 @@
+# Container Queries
+
+<p class="description">Material UI provides a utility function for creating CSS container queries based on theme breakpoints.</p>
+
+## Usage
+
+To create [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries), use `theme.containerQueries` with any method available in the [`theme.breakpoints`](/material-ui/customization/breakpoints/#api).
+The value can be unitless (in which case it'll be rendered in pixels), a string, or a breakpoint key. For example:
+
+```js
+theme.containerQueries.up('sm'); // => '@container (min-width: 600px)'
+```
+
+:::info
+One of the ancestors must have the CSS container type specified.
+:::
```

Comment:
I'd move this below the demo—it feels like a note that makes sense after you've explored the demo's code.